framework/include/interfaces/Coupleable.h

```diff
@@ -267,6 +267,33 @@ class Coupleable
   std::vector<const VariableValue *> coupledVectorTagValues(const std::string & var_name,
                                                             const std::string & tag_name) const;
 
+  /**
+   * Returns value of a coupled array variable for a given tag
+   * @param var_name Name of coupled array variable
+   * @param tag vector tag ID
+   * @param comp Component number for vector of coupled variables
+   * @return Reference to a VariableValue for the coupled array variable
+   * @see Kernel::_u
+   */
+  virtual const ArrayVariableValue &
+  coupledVectorTagArrayValue(const std::string & var_name, TagID tag, unsigned int comp = 0) const;
+
+  virtual const ArrayVariableValue & coupledVectorTagArrayValue(const std::string & var_name,
+                                                                const std::string & tag_name,
+                                                                unsigned int comp = 0) const;
+
+  /**
+   * Returns the values for all of a coupled variable's components for a given tag
```

Comment:
```suggestion
   * Returns the values for all the coupled variables desired for a given tag
```

Reply:
same for the others. 
The docstrings say component but it doesnt mean variable component, it means different variables in the string